@ljacobson64 seems like these changes don't work with the f95 compiler travis-ci is using:

```
CMakeFiles/gtol.dir/ensdf_processing/GTOL/gtol.f.o: In function `trapzd.1593':
gtol.f:(.text+0x4fc5): undefined reference to `real_'
CMakeFiles/gtol.dir/ensdf_processing/GTOL/gtol.f.o: In function `qromb.1599':
gtol.f:(.text+0x5184): undefined reference to `real_'
CMakeFiles/gtol.dir/ensdf_processing/GTOL/gtol.f.o: In function `fndlmt.1607':
gtol.f:(.text+0x5532): undefined reference to `real_'
gtol.f:(.text+0x55bc): undefined reference to `real_'
collect2: ld returned 1 exit 
```

--- a/src/ensdf_processing/GTOL/gtol.f
+++ b/src/ensdf_processing/GTOL/gtol.f
@@ -4093,7 +4093,8 @@ c         END IF
 !
 !     FUNCTIONS USED
 !
-      REAL(KIND=8), INTRINSIC :: DBLE, REAL
+      REAL(KIND=4), INTRINSIC :: REAL
+      REAL(KIND=8), INTRINSIC :: DBLE
 !
 !     Local variables
 !

--- a/src/ensdf_processing/GTOL/gtol.f
+++ b/src/ensdf_processing/GTOL/gtol.f
@@ -3944,8 +3944,7 @@ c         END IF
 !
 !     Functions used
 !
-      REAL(KIND=4), INTRINSIC :: ABS
-      REAL(KIND=4), EXTERNAL :: REAL
+      REAL(KIND=4), INTRINSIC :: ABS, REAL
 !
 !     Dummy arguments
 !
@@ -3960,12 +3959,12 @@ c         END IF
       SIGma = Dx
       b2 = MU + 1.28*SIGma
       high = MU + 10.0*SIGma
-      CALL QROMB(REAL(0.0),high,denom)
+      CALL QROMB(REAL(0.0,4),high,denom)
       low = MU + 1.28*SIGma
       IF(low.LE.0.0) low = 5.0*SIGma
       counts = 1
       DO WHILE (.TRUE.)
-         CALL QROMB(REAL(0.0),low,area)
+         CALL QROMB(REAL(0.0,4),low,area)
          prob = area/denom
          IF((prob.GE.0.899.AND.prob.LE.0.901).OR.counts.GT.100) THEN
             b1 = low
@@ -4040,9 +4039,8 @@ c         END IF
 !     Functions used
 !
       INTEGER(KIND=4), INTRINSIC :: INT
-      REAL(KIND=4), INTRINSIC :: ABS, ALOG10
+      REAL(KIND=4), INTRINSIC :: ABS, ALOG10, REAL
       REAL(KIND=8), INTRINSIC :: DABS, DBLE
-      REAL(KIND=4), EXTERNAL :: REAL
 !
 !     Local variables
 !
@@ -4061,7 +4059,7 @@ c         END IF
          jj = j
          CALL TRAPZD(A,B,s(j),jj)
          IF(j.GE.k) THEN
-            CALL POLINT(h(j-km),s(j-km),k,REAL(0.),Ss,dss)
+            CALL POLINT(h(j-km),s(j-km),k,REAL(0.,4),Ss,dss)
 !           Add check for when integral is zero
             IF(Ss.EQ.0..AND.dss.EQ.0.) RETURN
 !           IF(ABS(DSS) .LE. EPS*ABS(SS))RETURN
@@ -4095,8 +4093,7 @@ c         END IF
 !
 !     FUNCTIONS USED
 !
-      REAL(KIND=4), EXTERNAL :: REAL
-      REAL(KIND=8), INTRINSIC :: DBLE
+      REAL(KIND=8), INTRINSIC :: DBLE, REAL
 !
 !     Local variables
 !
@@ -4112,7 +4109,7 @@ c         END IF
          x = A + 0.5D+0*del
          dsum = 0.D+0
          DO j = 1, it
-            dsum = dsum + DBLE(NORMFUNC(REAL(x)))
+            dsum = dsum + DBLE(NORMFUNC(REAL(x,4)))
             x = x + del
          END DO
          S = 0.5D+0*(S+(DBLE(B)-DBLE(A))*dsum/tnm)

